Dockerfile.spark

```diff
@@ -0,0 +1,2 @@
+FROM godatadriven/spark:2.4.5
+EXPOSE 10000/tcp
```

Comment:
The EXPOSE command doesn't really actually do anything. 
It's a documentation feature, see https://docs.docker.com/engine/reference/builder/#expose
So I wouldn't create a dockerfile just to add it.

Reply:
Btw, I'll add a 2 tag to this image such that we can refer to the latest spark 2 image instead of fixing it to a particular minor release

Reply:
I think circleci needs it to figure out which ports to bind, but I'll double check

also I don't want to have that Dockerfile in the final PR, I was planning on getting that change to your dockerhub, see the TODO on the PR comment

Reply:
you were right, had something wrong at some point with the ports and I must have fixed the actual issue and assumed it was the EXPOSE statement, thanks for the explanation